### ISSUE TITLE
mqtt als api nutzen

### ISSUE THREAD

**Issue by thoka** [HashTec-Potsdam/wer-ist-da#1]
Wie wäre es, wenn der aktuelle Status über mqtt veröffentlicht würde ?
Um Weiterverteilung, Benachrichtigung etc könnten sich dann erprobte Tools kümmern.

**Comment by niccokunzmann** [comment]
Ja, das klingt gut. Welche Tools gibt es und welche würdest du nutzen? Gibts sowas wie Telegram-Chat oder so?
Beitragen kannst du das auch gern.
